### DISCUSSION TITLE
GNNExplainer coefffs meaning

### DISCUSSION THREAD

**Discussion by DogitoErgoSum** [pyg-team/pytorch_geometric#5832] [Q&A]
![image](https://user-images.githubusercontent.com/51344466/198139317-1b102dcd-044d-4cff-8683-a780bac6f691.png)

What is the meaning of coeffs["node_feat_ent"] and  coeffs["node_feat_size"]?

**Comment by rusty1s** [comment]
It denotes the reduction to use when computing the loss over node feats. By default, we average the losses across features.
